lep/DRAFT/010_Super_Easy_User_Experience.md

```diff
@@ -0,0 +1,217 @@
+1. Lantern starts up  and adds itself to the system start-up sequence.
+
+### Ubuntu Linux
+
+1. The user visits https://getlantern.org.
+1. If the user is on a desktop computer, the website displays a "Download"
+   button that triggers a `.deb` download for 386 or amd64.
+1. When the user is on a mobile device, she can provide her e-mail and a
+   special service will send Lantern download links to her inbox.
+1. The user downloads the `.deb` file. A new file named `lantern-installer.deb`
+   is created in the `Downloads` folder.
+1. The user double-clicks on the `lantern-installer.deb` file and the Software
+   Installer shows the contents.
+1. The user clicks "Install".
+1. The user starts Lantern by going to the Applications directory and looking
+   for the Lantern icon or by typing "Lantern".
+1. Lantern starts up  and adds itself to the system start-up sequence.
+
+## Current usage
+
+## Proposal
+
+This is the current download flow and I think it's great as it is:
+
+1. The user visits https://getlantern.org.
+1. If the user is on a desktop computer, the website displays a "Download"
+   button that triggers the downloading of the installer. This installer is
+   different for each operating system.
+1. When the user is on a mobile device, she can provide her e-mail and a
+   special service will send Lantern download links to her inbox.
+
+### Windows
+
+The Windows installation proccess is currently rather easy: no questions asked,
+but also no message tells the user that Lantern is installed or how to open
+Lantern again so it may be a bit confusing too.
+
+In order to potentially avoid confusing users I propose a more classical
+installation that asks for a confirmation before installing and that tells the
+user that Lantern has been installed afterwards.
+
+1. The user downloads the `.exe` file. A new file named `lantern-installer.exe`
+   is created in the `Downloads` folder.
+1. The user double-clicks the `lantern-installer.exe` file.
+1. The installation shield asks for confirmation. "Do you want to install
+   Lantern 2.0.0 on this machine? Y/N".
+1. The Lantern installer shows up and a progress bar is filled. The
+   installation shield stops and a message "Lantern has been installed. Thank
+   you!" appears and requires a user action to be dismissed.
+1. The installation process also performs the following actions:
+  1. A new desktop shortcut is created.
+  1. A new start menu shortcut is created.
+  1. A new Lantern folder is created in the start menu.
+1. After dismissing the message, Lantern is opened.
+1. Lantern adds itself to the system start-up sequence.
+
+### OSX
+
+The OSX installation proccess would be a bit confusing for Windows users, but
+it's usual stuff for OSX users.
+
+I propose not to assume that the user that is going to install Lantern knows
+how to install something in OSX and has no idea about the meaning of the
+Applications directory. I also propose renaming the volume that is created by
```

Comment:
That doesn't quite make sense to me. They're an OSX user and not a Windows user by definition right? And dragging to the Applications directly is Apple's recommended approach, although it would be interesting to also consider getting it into the Mac Store.